### ISSUE TITLE
Route update causes issues

### ISSUE THREAD

**Issue by fahlmant** [openshift/operator-custom-metrics#11]
The route update  for an existing route is giving the following:
```
  status:
    ingress:
    - conditions:
      - lastTransitionTime: 2019-07-23T13:47:57Z
        message: route pagerduty-operator already exposes pagerduty-operator-pagerduty-operator.c39b.hive-stage.openshiftapps.com
          and is older
        reason: HostAlreadyClaimed
        status: "False"
        type: Admitted
      host: pagerduty-operator-pagerduty-operator.c39b.hive-stage.openshiftapps.com
      routerName: router
```
The route continues to exist but does not work.

**Comment by fahlmant** [comment]
The error relates to redundant services, not the route. Closing

**Comment by fahlmant** [comment]
Possibly related BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1723527

**Comment by fahlmant** [comment]
https://bugzilla.redhat.com/show_bug.cgi?id=1660598
https://access.redhat.com/errata/RHBA-2019:0326

Closing as it has been fixed in a z-stream update to Openshift 3.11
